### DISCUSSION TITLE
V3

### DISCUSSION THREAD

**Discussion by DBT85** [xZetsubou/ha-custom-sidebar#1] [General]
I just wondered if you wanted to just update this to be v3 since it is the only working version now and since the actual V2 has been archived by its owner.

**Comment by xZetsubou** [comment]
To be honest getting rid of version tag and go with `ha-custom-sidebar` is more sensible.

**Reply by DBT85** [reply]
Sounds great. The readme needs updating though for anyone coming in as right now its confusing as to what is what.

**Reply by ahmaddxb** [reply]
I get this error when try to add the repo

![image](https://github.com/xZetsubou/ha-custom-sidebar/assets/43728586/e3fbc047-b00a-46fe-8be9-5e03853b2652)

**Reply by xZetsubou** [reply]
@ahmaddxb Sorry, Should be fixed now.

**Reply by ahmaddxb** [reply]
Thanks
